### PR TITLE
Report protobuf design

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

File: src/main/proto/Report.proto [hunk 1]
```diff
@@ -65,24 +65,58 @@ message Schedule {
 message ReportLog {
     string task_status = 1;
     string task_executed = 2;
-    string threshold_met = 3;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto:70]: change it to camel notiation

File: src/main/proto/Report.proto [hunk 2]
```diff
@@ -65,24 +65,58 @@ message Schedule {
...
-    string notification_sent = 4;
+    bool threshold_met = 3;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: change it to camel notiation

File: src/main/proto/Report.proto [hunk 3]
```diff
@@ -65,24 +65,58 @@ message Schedule {
...
+}
+
+message TeamConfigList {
+    int64 success = 1;
+    repeated TeamConfigParameters records = 2;
+}
+
+message EmailConfigList {
+    int64 success = 1;
+    repeated EmailParameters records = 2;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: obejcts name should be emailParameters instead of records

File: src/main/proto/Report.proto [hunk 4]
```diff
@@ -65,24 +65,58 @@ message Schedule {
...
+    string host = 1;
+    string sender = 2;
+    int64 port = 3;
+    string user = 4;
+    string password = 5;
+}
+
+message TeamConfigList {
+    int64 success = 1;
+    repeated TeamConfigParameters records = 2;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: rename records to teamConfigParameters

File: src/main/proto/Report.proto [hunk 5]
```diff
@@ -65,24 +65,58 @@ message Schedule {
...
-    string threshold_met = 3;
-    string notification_sent = 4;
+    bool threshold_met = 3;
+    bool notification_sent = 4;
     string channel = 5;
 }
 
 message ChannelConnection {
-    repeated Channelparameters ChannelProperties = 1;
+    repeated Channelparameters channelProperties = 1;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: https://github.com/viz-centric/flair-notifications/pull/98/files

File: src/main/proto/Report.proto [hunk 6]
```diff
@@ -65,24 +65,58 @@ message Schedule {
...
 }
 
 message Channelparameters {
   string id = 1;
-  repeated ConnectionProperties connectionProperties = 2;
+  repeated ConnectionProperties channelParameters = 2;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: https://github.com/viz-centric/flair-notifications/pull/98/files

**Review by johibkhan2 — DISMISSED** [review]
1)check the build mvn clean install
2)test it with flair-bi locally

File: src/main/proto/Report.proto [hunk 7]
```diff
@@ -65,24 +65,57 @@ message Schedule {
...
-    string threshold_met = 3;
-    string notification_sent = 4;
+    bool thresholdMet = 3;
+    bool notificationSent = 4;
     string channel = 5;
 }
 
 message ChannelConnection {
-    repeated Channelparameters ChannelProperties = 1;
+    repeated ChannelParameters channelProperties = 1;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: channelProperties to be renamed channelParameters

**Review by johibkhan2 — DISMISSED** [review]

File: pom.xml [hunk 8]
```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.project.bi</groupId>
     <artifactId>flair-messages</artifactId>
-    <version>0.3.1-SNAPSHOT</version>
+    <version>0.2.2-SNAPSHOT</version>
```
Comment (johibkhan2) [inline pom.xml]: please do not change the version of

File: src/main/proto/ReportService.proto [hunk 9]
```diff
@@ -77,23 +77,66 @@ message SearchReportsResponse {
 }
 
 message GetChannelPropertiesRequest {
-
+    
 }
 
 message GetChannelPropertiesResponse {
-    repeated ChannelConnection ChannelConnection = 1;
+   repeated ChannelConnection request = 1;
```
Comment (johibkhan2) [inline src/main/proto/ReportService.proto]: please remove repeat from here

**Review by johibkhan2 — APPROVED** [review]

**Review by johibkhan2 — APPROVED** [review]

File: src/main/proto/Report.proto (hunk 1 above)
Comment (ir73) [reply src/main/proto/Report.proto:70]: I think the official naming convention suggests to use snake case https://developers.google.com/protocol-buffers/docs/style?csw=1

**Pull request merged**